src/site/_includes/content/payments/browser-compatibility.njk

```diff
@@ -18,19 +17,17 @@ status depends on the browser.
       <td></td>
       <td>Desktop</td>
       <td>Android</td>
-      <td>iOS</td>
       <td>Desktop</td>
       <td>Mobile</td>
-      <td></td>
+      <td>Desktop/Mobile</td>
     </tr>
     <tr>
       <td>Payment Request API</td>
       <td>✔</td>
       <td>✔</td>
```

Comment:
I can, but it can't tell:
* Payment Handler API compatibility because it's not documented.
* The availability with native payment apps.
* The restriction Safari has with Apple Pay

I think it's fine the way it is now.